### PR TITLE
impr(quotes): add Kazakh quotes (@w0ik_id)

### PR DESCRIPTION
**Pull request by w0ikid**
### Description
- added 26 Kazakh quotes 

1. > 'The easiest thing in the world is to give advice to someone, the hardest thing is to understand yourself.'

2. > 'External beauty must be enlivened by internal beauty to be complete.'

3. > ''Valuing words, adhering to words, and seeking meaning in them makes a person more cultured.

4. > 'Don’t rush ahead to be better than others, know your limits. Be content when you are equally matched. Your ancestors said, ‘Contentment feeds you, but discontentment will cost you your only horse.’ But don’t tolerate being trampled; never let yourself be walked on.'

5. > 'Stay away from people who rush ahead when taking but hide at the back when giving, who meticulously count what they’ve taken and given. Freeing yourself from such a person is already a great gain.'

6. > 'The goal of upbringing is not to superficially label someone as human, but to truly make them human in the true sense.'

7. > 'The first condition for the proper development of a child's language is not to speak to the child in broken words.'

8. > 'To truly be human in the true sense, one must not only love oneself and one's loved ones but also love humanity as a whole.'

9. > 'The Kazakh nation is a lazy nation. A nation left behind by time. But time keeps moving forward even as you keep walking around saying you’re lazy.'

10. > 'Believe in yourself, bring yourself along, with your labor and your intellect both contributing.'

11. > 'Smart, noble, honest—these days, many people don’t praise anyone.'

12. > 'Keep your intellect, strength, and heart united, and then you will be complete apart from others.'

13. > 'A sincere person loves their homeland, to a sincere, pure heart, the homeland is like a mother. Serving the homeland brings joy and pride. The love for the homeland, especially the love we have, cannot be measured.'

14. > 'If you are a hero but don’t wield a weapon against the enemy, what is the use of heroism? If you are rich but don’t benefit your people, what is the use of your wealth?'

15. > 'By raising sons, we educate the people; by raising daughters, we educate the nation.'

16. > 'Although many people are given intellect, not everyone is granted a heart equally.'

17. > 'Blessings and prosperity—your father is precious, your loving mother is precious. Your kind sister is precious. Your lofty mountain is your precious father.'

18. > 'A stubborn person is difficult to tame; a brave young man does not shy away from stubbornness.'

19. > 'Some people are more loyal than a dog, they do not forget friendship and serve faithfully.'

20. > 'What’s the use of reading and stuffing the words of sages into your mind if you don’t apply them in practice?'

21. > 'Science is the centuries-long work of human thought that has gathered all the phenomena of the surrounding world through systematization.'

22. > 'Disentangling people’s misunderstandings is even harder than me splitting atoms.'

23. > 'First, master the rules of the game well. Only then should you start trying to play better than others.'

24. > 'Any endeavor has three conditions for success: first, intention; then, effort; and then, discipline.'

25. > 'Goodness that comes through struggle and patience is truly a precious gain.'

26. > 'The nature of the times: those with much knowledge do not understand the people, and those with much wealth are dominated by the people.'

### Checks

- [X] Adding quotes?
- [X] Make sure to include translations for the quotes in the
description (or another comment) so we can verify their content.

### CODE REVIEW

**Comment by Miodec** [comment]
Could you please add more quotes? I woudl probably start a new file with at least 20-30 quotes to reduce repetition.

File: frontend/static/quotes/kazakh.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        "id": 2
+      },
+      {
+        "text": "Сөзді бағалау, сөзге тоқтау, сөзден мағына іздеу адамды мәдениеттендіре түседі.",
+        "source": "І. Омаров",
+        "length": 79,
+        "id": 3
+      },
+      {
+        "text": "Өзгеден артық боламын деп арындама,шамаңды шақта. Тең түскенге қанағат ет. “Қанағат қарын тойғызар,қанағатсыздық жалғыз атын сойғызар” деген ата-бабаң.Бірақ тепкіге төзбе, ешқашан аяққа басылмағын.",
```
Comment (Miodec) [inline frontend/static/quotes/kazakh.json:26]: Shouldnt there be a space here? `ата-бабаң.Бірақ` > `ата-бабаң. Бірақ`

File: frontend/static/quotes/kazakh.json [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        "id": 12
+      },
+      {
+        "text": "Адал адам Отанын сүйеді, адал, ақ жүрекке Отан — анасындай. Отанның дегенін істеу — қуаныш, мақтаныш. Отанға деген махаббатты, әсіресе біздегі махаббатты өлшеуге болмайды.",
+        "source": "Баубек Бұлқышев",
+        "length": 167,
+        "id": 13
+      },
+      {
+        "text": "Батыр боп жауға найзаң тимесе, Батырлықтан не пайда. Бай болып халқыңа пайдаң тимесе, Байлығыңнан не пайда",
```
Comment (Miodec) [inline frontend/static/quotes/kazakh.json:86]: Missing period at the end?

Comment (w0ikid) [reply frontend/static/quotes/kazakh.json]: fixed

File: frontend/static/quotes/kazakh.json (hunk 1 above)
Comment (w0ikid) [reply frontend/static/quotes/kazakh.json]: yep fixed

**Pull request closed without merging**